Cargo.lock

```diff
@@ -53,6 +55,15 @@ version = "0.3.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7ebdb29d2ea9ed0083cd8cece49bbd968021bd99b0849edb4a9a7ee0fdf6a4e0"
 
+[[package]]
+name = "hybrid-array"
+version = "0.2.0-rc.5"
```

Comment:
a `cargo update` will bump that to `0.2.0-rc.6` and that will break the tree. I'm not entirely sure how that is possible,
I think the version pinning over at `crypto-common` is too loose for an rc?

Reply:
It's a regression I still need to fix.

It's version requirement is loose because otherwise each release requires a release of every other downstream crate which is somewhat unmanagable.